### ISSUE TITLE
Better style (Help Wanted)

### ISSUE THREAD

**Issue by Jak2k** [Jak2k/Project-Competentia#7]
We needabetter design...

**Comment by Naainz** [comment]
Do you still need help, because I can contribute (this is just a bit old so I don't know xD)

**Comment by Jak2k** [comment]
Yes, I would definitely appreciate your contribution. Thank you for reaching out and offering to help.

**Comment by Naainz** [comment]
Would you mind if I try and rework your current design with HTML code, because it is better for the type of project you are looking for

**Comment by Jak2k** [comment]
What do you mean with HTMLcode? The templates are just HTML + Some syntax for putting content inside. While rendering they are the rendered to plain HTML.

**Comment by Naainz** [comment]
They are rendered to show plain HTML, but with HTML you can actually have much better & nicer designs than Markdown.

**Comment by Jak2k** [comment]
The actual layout is HTML. You can find it in ˋ/layouts/_defaultˋ. The markdown is sanitized, converted and injected in the places, which are defined in the layouts. The styles are located in ˋ/static/stylesˋ.

The way the text is rendered is acctually ok. Just the layout is really bad.
